Commands/ComplexCommands/TutorialWaitResponse/OtherWaitResponse.go

```diff
@@ -0,0 +1,117 @@
+customize/add more questions, along the way I also added a responses logger
+This CC is also an absolute fucking mess with the styling, it's a slash of DZ's and My own (they don't mix)
+*/}}
+
+{{/* CONFIGURATION VALUES START*/}}
+{{$trigger:=`-start`}} {{/* the trigger is set to -trigger, basic regex knowledge is required to edit this */}}
+{{$timer:=30}} {{/* 60s timer, if timer ends with no response, the command is automatically disabled */}}
+{{/* QUESTIONS
+    regex: regex for the response you want for that question (same sdict)
+    title: subject/title of that embed
+    description: description
+*/}}
+{{$q:=cslice
+  	(sdict "regex" `\A\d+(\s+|\z)` "title" "Number" "description" "type a number")
+  	(sdict "regex" `\A.+(\s+|\z)` "title" "Anything" "description" "type anything")
+  	(sdict "regex" `\A(happy|sad)(\s+|\z)` "title" "Anything" "description" "are you happy or sad")
+}}
+{{/* DO NOT EDIT BELOW (unless you want to learn how it works or know what you're doing c: ) */}}
+  
+{{$databaseValue:=toInt (dbGet .User.ID "waitResponse").Value}}
+{{$dontChangeStage:=0}}
+{{$changeStage:=0}}
+{{ $responses := 0 }}
+{{$colour:=0x4B0082}}
+{{$position:=0}}
+{{ $q = $q.Append (sdict "regex" `.*` "title" "Responses" "description" "_ _") }}
+
+{{define "err"}}
+	{{ $id := sendMessageRetID nil "Incorrect value please try again." }}
+	{{ deleteMessage nil $id 3 }}
+	{{ deleteTrigger 0 }}
+{{end}}
+
+{{/* sets colour to role colour */}}
+{{range .Guild.Roles}}
+	{{- if and (in $.Member.Roles .ID) .Color (lt $position .Position)}}
+		{{- $position =.Position}}{{$colour =.Color}}
+	{{- end -}}
+{{end}}
+
+{{/* sets default value for $embed */}}
+{{$embed:=sdict "author" (sdict "icon_url" (.User.AvatarURL "256") "name" (print "User: " .User.String)) "footer" (sdict "text" "Type cancel to cancel the questionaire.") "color" $colour}}
+
+{{/* checks if it was executed by scheduleUniqueCC */}}
+{{if .ExecData}}
+	{{$embed.Set "title" "Timed Out!"}}
+	{{$embed.Set "description" (print "You have not entered a response after " $timer " seconds. As such, the commmand has been cancelled.")}}
+	{{sendMessage nil (cembed $embed)}}
+{{else}}
+
+	{{ range $num, $qval := $q }}
+{{/* Starting*/}}
+		{{ if reFind (print `\A(?i)` $trigger `(\s+|\z)`) $.Message.Content }}
+			{{ if and (not $databaseValue) (not $num) }}
+				{{ $embed.Set "title" $qval.title }}
+				{{ $embed.Set "description" $qval.description }}
+				{{ dbSetExpire $.User.ID "responses" (cslice) $timer }}
+				{{$changeStage =1}}
+			{{ end }}
+		{{ else if and $num $databaseValue }}
+			{{ $responses = (cslice).AppendSlice (dbGet $.User.ID "responses").Value }}
+		{{ end }}
+
+{{/* Cancellation*/}}
+		{{ if and (eq (lower $.Message.Content) "cancel" "quit" "stop") $num }}
+			{{$embed.Set "title" "Cancelled"}}
+			{{$embed.Set "description" (print (or $.Member.Nick $.User.Username) "#" $.User.Discriminator " has decided to cancel the questionaire.")}}
```

Comment:
erm... remove # discrim part... nickname with discrim is rather odd, altho (or .Member.Nick .User.String) is fine ig, altho I'd prefer to just keep username & nickname combo

Reply:
Fixed

Reply:
It's now just Username#Discrim